.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: check-merge-conflict
+    -   id: debug-statements
+    -   id: mixed-line-ending
+    -   id: trailing-whitespace
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py37-plus"]
+-   repo: https://github.com/adamchainz/django-upgrade
+    rev: '1.4.0'
+    hooks:
+      - id: django-upgrade
+        args: [--target-version, "2.2"]
```

Comment:
I think this is one of the things you asked about on Slack and its a bit easier to respond here.
This version is the minimum supported version. This tool will optimise the Django specific code in a commit, using the features available in Django 2.2.